lib/sgtn-client/loader/source_comparer.rb

```diff
@@ -11,7 +11,7 @@ def load_bundle(component, locale)
 
     translation_bundle_thread = Thread.new { super(component, locale) }
     old_source_bundle = super(component, SgtnClient::LocaleUtil.get_source_locale)
-    source_bundle = get_cs(component, SgtnClient::LocaleUtil.get_source_locale)
+    source_bundle = SgtnClient::Source.load_bundle(component)
```

Comment:
This does not seem to utilize the cache for source bundle.

Reply:
get_cs is out of loader, can use it.